client/README.md

```diff
@@ -0,0 +1,24 @@
+# Development guide
```

Comment:
This MR is ready for review, this is what it looks like currently:

- Note that some UI element is incomplete (like navigation dropdown), since there is no design for it yet. I'll update it along the way when I work on other issue

[Screencast from 08-02-2025 09:31:16 AM.webm](https://github.com/user-attachments/assets/bef3b75e-e6e3-406b-a97b-3cc4c74b69fc)